src/game/server/gamemodes/base_pvp/player.cpp

```diff
@@ -99,6 +99,12 @@ void CPlayer::InstagibTick()
 		m_FastcapQueryResult = nullptr;
 	}
 
+	//Dont keep rollback enabled if server does not allow it
+	if(!g_Config.m_SvAllowRollback)
+	{
+		m_RollbackEnabled = false;
```

Comment:
So rollback is only deactivated on round end? Would be nice if there was a config chain. And setting the rollback config would apply instantly for all players.